gateway/1.4/modules/ROOT/pages/index.adoc

```diff
@@ -37,7 +37,7 @@ The gateway runtime is a customer-hosted distributed entity installed in any clo
 FluentBit implementation enables log output to local files, or to aggregators such as New Relic, Sumo Logic, and Splunk.
 
 == Replicas
-A runtime unit is called a `replica`. While a single runtime can support multiple backend APIs for high availability, MuleSoft recommends installing Flex Gateway as a cluster, which is a Kubernetes Ingress controller with multiple replicas running in parallel. Using clusters increases gateway performance and robustness, and is recommended for high-performance applications that must scale.
+A runtime unit is called a `replica`. While a single runtime can support multiple backend APIs for high availability, MuleSoft recommends deploying Flex Gateway as a cluster with multiple replicas running in parallel. Using clusters increases gateway performance and robustness, and is recommended for high-performance applications that must scale.
```

Comment:
```suggestion
A runtime unit is called a `replica`. While a single runtime can support multiple backend APIs for high availability, it is best to deploy Flex Gateway as a cluster with multiple replicas running in parallel. Using clusters increases gateway performance and robustness, and is recommended for high-performance applications that must scale.
```

Reply:
I freaking hate "robustness," but the main goal was to get rid of references to _ingress controller_. So I resisted rewording.